src/python/CMakeLists.txt

```diff
@@ -69,5 +69,10 @@ TARGET_LINK_LIBRARIES(_createrepo_c
                         ${PYTHON_LIBRARY}
                      )
 
-INSTALL(FILES __init__.py DESTINATION ${PYTHON_INSTALL_DIR}/createrepo_c)
-INSTALL(TARGETS _createrepo_c LIBRARY DESTINATION ${PYTHON_INSTALL_DIR}/createrepo_c)
+IF (SKBUILD)
+    INSTALL(FILES __init__.py DESTINATION src)
+    INSTALL(TARGETS _createrepo_c LIBRARY DESTINATION src)
```

Comment:
This bit is not ideal.  I ran into issues using a different directory, and I'm not sure yet whether it's because I was doing something wrong or if it's a bug w/ scikit-build.  I'll keep investigating.

It's not really a huge issue either, though.